Source/Urho3D/Network/Connection.h

```diff
@@ -98,27 +90,30 @@ struct PackageUpload
     unsigned totalFragments_;
 };
 
+class Connection;
+
 /// %Connection to a remote network host.
 class URHO3D_API Connection : public AbstractConnection
 {
     URHO3D_OBJECT(Connection, AbstractConnection);
+    friend class Network;
 
 public:
     using AbstractConnection::SendMessage;
 
     /// Construct with context, RakNet connection address and Raknet peer pointer.
-    Connection(Context* context);
+    Connection(Context* context, NetworkConnection* wtConnection = nullptr);
```

Comment:
what is "wt"?

Reply:
It is an artifact from previous iterations. "wt" stands for "web transport". But eventually it turned out that this wont necessarily have anything to do with "web" so yeah, its just a transport now. Ill remove these.